model/work.py

```diff
@@ -1545,23 +1545,37 @@ def query_to_json_array(query):
         # The work quality field is stored in the main document, but
         # it's also stored here, so that we can apply a nested filter
         # that combines quality with other fields found only in the subdocument.
+
+        def explicit_bool(label, t):
+            # Ensure we always generate True/False instead of
+            # True/None. Elasticsearch can't filter on null values.
+            return t.label(label)
+            return case([(t, True)], else_=False).label(label)
```

Comment:
You're not getting to line 1553 because of the return statement in the previous line.

Reply:
Ugh, that's my test code trying to reproduce the error. It needs to go.